packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -52,64 +52,69 @@ interface CheckboxProps {
   onChange?(newValue: boolean): void;
 }
 
-export function Checkbox({
-  checked,
-  defaultChecked,
-  disabled,
-  label,
-  name,
-  value,
-  indeterminate = false,
-  description,
-  onChange,
-}: CheckboxProps) {
-  const wrapperClassName = classnames(
-    styles.wrapper,
-    disabled && styles.disabled,
-  );
-  const inputClassName = classnames(styles.input, {
-    [styles.indeterminate]: indeterminate,
-  });
-  const iconName = indeterminate ? "minus2" : "checkmark";
+export const Checkbox = React.forwardRef<HTMLDivElement, CheckboxProps>(
+  (
+    {
+      checked,
+      defaultChecked,
+      disabled,
+      label,
+      name,
+      value,
+      indeterminate = false,
+      description,
+      onChange,
+    },
+    ref,
```

Comment:
We are using `React.forwardRef` here in order to allow our checkbox to be used as a reference for our popover component